src/main/java/mate/jdbc/exception/DataProcessingException.java

```diff
@@ -0,0 +1,7 @@
+package mate.jdbc.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String massage, Throwable e) {
+        throw new RuntimeException(massage, e);
```

Comment:
why do we throw RuntimeException in constructor of DataProcessingException? 
```suggestion
        super(message, e);
```